external/llvm-project/mlir/test/lit.site.cfg.py.in

```diff
@@ -51,10 +51,10 @@ config.enable_vulkan_runner = @MLIR_VULKAN_RUNNER_ENABLED@
 config.enable_bindings_python = @MLIR_BINDINGS_PYTHON_ENABLED@
 config.mlir_integration_test_dir = "@MLIR_INTEGRATION_TEST_DIR@"
 config.intel_sde_executable = "@INTEL_SDE_EXECUTABLE@"
-config.mlir_run_amx_tests = "@MLIR_RUN_AMX_TESTS@"
-config.mlir_run_x86vector_tests = "@MLIR_RUN_X86VECTOR_TESTS@"
-config.mlir_run_cuda_tensor_core_tests = "@MLIR_RUN_CUDA_TENSOR_CORE_TESTS@"
-config.mlir_include_integration_tests = "@MLIR_INCLUDE_INTEGRATION_TESTS@"
+config.mlir_run_amx_tests = @MLIR_RUN_AMX_TESTS@
```

Comment:
If these guys are quoted, we end up with "0" which is interpreted as True when we want False.